app/controllers/vaccination_records_controller.rb

```diff
@@ -13,14 +13,15 @@ def show
   end
 
   def export_dps
-    send_data(dps_export.export!, filename: dps_export.filename)
+    send_data(dps_export.csv, filename: dps_export.filename)
   end
 
   def reset_dps_export
-    dps_export.reset!
+    campaign.dps_exports.each { _1.vaccination_records = [] }
```

Comment:
I don't think this will commit the change to the database, you'll likely need to call `save` somewhere.

Reply:
This actually saves the change immediately. I did double-check this, as I wasn't quite 100% sure myself. I have this vague notion that maybe this is configurable somewhere, but also this semantic kind of makes sense since there's no change directly on the `DPSExport` record to save.

<img width="896" alt="image" src="https://github.com/user-attachments/assets/59d69777-2185-4752-97b7-b845cb29d671">

Reply:
Oh interesting, I wouldn't have expected that!